jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/DeploymentManager.java

```diff
@@ -393,7 +393,7 @@ private void fail(TrackedContext tracked)
         }
     }
 
-    private TrackedContext startTracking(ContextHandler contextHandler)
+    protected TrackedContext startTracking(ContextHandler contextHandler)
```

Comment:
Don't worry about these modifier changes, this just exists to make the code in jetty-deploy happy for this WIP.